lease/lessor.go

```diff
@@ -491,6 +492,7 @@ func (le *lessor) initAndRecover() {
 			itemSet: make(map[LeaseItem]struct{}),
 			expiry:  forever,
 			revokec: make(chan struct{}),
+			fresh:   true,
```

Comment:
probably this can be a flag on lessor itself instead? we change the extend accordingly. i do not feel we should put the calculation logic into lease.